build/kickstarts/base.ks.erb

```diff
@@ -20,6 +20,7 @@ repo --name=SCL --baseurl=http://mirror.centos.org/centos/6/SCL/x86_64/
 repo --name=manageiq-scl --baseurl=http://copr-be.cloud.fedoraproject.org/results/jrafanie/manageiq-scl/epel-6-x86_64/
 repo --name=ipa --baseurl=http://copr-be.cloud.fedoraproject.org/results/adelton/identity_demo/epel-6-x86_64/
 repo --name=certmonger --baseurl=http://copr-be.cloud.fedoraproject.org/results/nalin/certmonger/epel-6-x86_64/
+repo --name=ipa_scl --baseurl=http://copr-be.cloud.fedoraproject.org/results/adelton/identity_demo_scl/epel-6-x86_64/
```

Comment:
@jrafanie @abellotti 
Which version of CentOS do we build on? The one that I was testing with was 6.6.